data/campaigns/Eastern_Invasion/scenarios/16_Eleventh_Hour.cfg

```diff
@@ -0,0 +1,2237 @@
+                                        {NEW_GOLD_CARRYOVER 40}
+                                    [/endlevel]
+                                [/command]
+                            [/option]
+                            [option]
+                                label= _ "defeat Mal-Ravanal in pitched battle. <i>(challenging)</i>"
+                                [command]
+                                    [message]
+                                        speaker=Owaec
+                                        message= _ "Excellent, I will begin preparing our armies!"
+                                    [/message]
+                                    [endlevel]
+                                        result=victory
+                                        next_scenario=17b_All-In
+                                        {NEW_GOLD_CARRYOVER 40}
+                                    [/endlevel]
+                                [/command]
+                            [/option]
+                        [/message]
+                    [/else]
+                [/if]
+            [/then]
+
+            [else]
+                [if]
+                    [have_unit]
+                        id=Gweddry
+                    [/have_unit]
+
+                    #--------------------
+                    # OWAEC DEAD, GWEDDRY ALIVE
+                    #--------------------
+                    [then]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "The city grows still at last. I wish there was time to mourn, but we still must decide how to deal with Mal-Ravanal."
+                        [/message]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "Owaec had wished for us to strike at the undead and isolate Mal-Ravanal by force of arms. With their armies in disarray after the failed attack on Weldyn, this may be our best opportunity."
+                        [/message]
+                        [message]
+                            speaker=Dacyn
+                            message= _ "I will not put the fate of all Irdya in the hands of Weldyn’s feeble defenders! I must face Mal-Ravanal in a duel, and I must be the one to strike the infernal lich down! I will trust none other than myself with such responsibility."
+                        [/message]
+                        [message]
+                            speaker=Konrad
+                            message= _ "Both of you alike speak wisdom. This is a difficult decision, and I have little time in which to make it."
+                        [/message]
+                        [message]
+                            speaker=Konrad
+                            message= _ "Lieutenant Gweddry, you are familiar with the situation, and you knew both Owaec and Dacyn well. Whose plan do you suggest I follow?"
```

Comment:
I suppose Gweddry is only speaking on behalf of the fallen Owaec and not putting forth the attack-in-force option as his own, but still seems a little odd for him to raise Owaec's opinion and then vote for Dacyn's choice anyway. Wonder if it could/should be rewritten so that Konrad makes the choice.